src/logics/compound-v2/tokens/data/cTokens.json

```diff
@@ -55,13 +55,6 @@
     "symbol": "cMKR",
     "name": "Compound Maker"
   },
-  "cREP": {
```

Comment:
@chouandy I found this file did not have `cWBTC2` info. Do we need to add it back? (I was checking the token info with `src/logics/compound-v2/scripts/get-tokens-data.ts`)

Reply:
I think it's OK. But to address unrelated questions or topics, it's often more appropriate to create a new issue in the repository. This helps keep discussions organized and ensures that issues and PRs are dedicated to their respective purposes. cc @zodahu

Reply:
Okay. Will create another ticket to track this issue.